build/Targets/Imports.targets

```diff
@@ -479,7 +479,7 @@
       />
 
     <Error 
-      Text="The 2.0 SDK is required to build this repo"
+      Text="The 2.1 SDK is required to build this repo"
```

Comment:
Are we able to use a variable of something else here?

Reply:
I'm not sure. I'll give it a try.

Reply:
Ok, I was able to use the variable :-)
The truncation is not very elegant. Let me know if there is a better way. Thanks